packages/eslint-config/src/server.js

```diff
@@ -0,0 +1,40 @@
+import eslint from '@eslint/js';
+import prettier from 'eslint-config-prettier';
+import importPlugin from 'eslint-plugin-import';
+import globals from 'globals';
+import tseslint from 'typescript-eslint';
+
+export const react = tseslint.config(
+  {
+    ignores: [
+      '**/__mocks__',
+      '**/__snapshots__',
+      '**/node_modules',
+      '**/dist',
+      '**/package-lock.json',
+      '**/.vscode',
+    ],
+  },
+  importPlugin.flatConfigs.recommended,
+  eslint.configs.recommended,
+  tseslint.configs.recommended,
+  prettier,
+  {
+    files: ['**/*.{js,ts}'],
```

Comment:
Would it be safe to add esm or cjs file extensions?

Reply:
I don't think that it would hurt. I'll add them.